cql/the-basics-of-cql/cinchy-supported-functions/cinchy-user-defined-functions-udfs/table-valued-functions.md

```diff
@@ -13,9 +13,9 @@ SELECT u.*
 FROM tableUDF() u
 ```
 
-## 3. Creating a Table in a UDF <a href="#static-table" id="static-table"></a>
+## Create a Table in a UDF <a href="#static-table" id="static-table"></a>
 
-To generate a table within a UDF for use in CQL, a dataTable will need to be created in the same format as the default Cinchy JSON Saved Query response _(Image 1)._
+To generate a table within a UDF for use in CQL, you need to create a data table in the same format as the default Cinchy JSON Saved Query response _(Image 1)._
```

Comment:
Unsure if this is a data type `dataTable` or a typo.

Reply:
Think a typo